app/services/yoksis/v1/ozgecmis.rb

```diff
@@ -3,46 +3,46 @@
 module Services
   module Yoksis
     module V1
-      # client = Services::Yoksis::V1::Ozgecmis.new
+      # client = Services::Yoksis::V1::Ozgecmis.new(12345678912)
       class Ozgecmis
-        def initialize
-          @username = Rails.application.credentials.yoksis[:user]
-          @password = Rails.application.credentials.yoksis[:password]
+        %i[
+          get_arastirma_sertifka_bilgisi_v1
```

Comment:
tipo mu normal mi? `sertifka`

Reply:
Tipo ama YOKSIS'in tiposu :) endpoint'in ismi bu sekilde